### PR TITLE
Update cert-manager to 0.5.0

### LINKED ISSUE #405: Upgrade cert-manager to 0.4
https://github.com/jetstack/cert-manager/releases/tag/v0.4.0

### PR DESCRIPTION
**Pull request by jnummelin**
fixes #405 

WIP:

- [x] some specs
- [x] migrate also `ClusterIssuer` type(s) if found

### CODE REVIEW

File: addons/kured/resources/foo.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+kind: Role
```
Comment (jakolehm) [inline addons/kured/resources/foo.yml]: These files probably don't belong to this pr? 😄

Comment (jnummelin) [reply addons/kured/resources/foo.yml]: Definitely not. They look like some leftovers from PSP testing. I'll remove them

Comment (jnummelin) [reply addons/kured/resources/foo.yml]: done

**Comment by jnummelin** [comment]
@jakolehm WIP label removed, PTAL

File: addons/cert-manager/addon.rb [hunk 2]
```diff
@@ -20,5 +20,53 @@
...
+  }
+
+  install {
+    apply_resources
+
+    migrate_le_acme_issuers
+    migrate_le_acme_cluster_issuers
   }
+
+  PATCH_SPEC = {
```
Comment (jakolehm) [inline addons/cert-manager/addon.rb]: `PATCH_SPEC` -> `patch_spec` (uppercase will leak as a global)

File: addons/openebs/resources/23-storage-class.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: storage.k8s.io/v1
```
Comment (jakolehm) [inline addons/openebs/resources/23-storage-class.yml]: Does not belong here.
